### ISSUE TITLE
a11y - Default sidebar "plone site" - Focus and labels - cms ui

### ISSUE THREAD

**Issue by Wagner3UB** [plone/volto#5122]
**Describe the bug**
The default sidebar have lots of problems with accessibility, including the right use of 'focus,' lack of 'aria-label' on checkboxes and selects or the aria-labeled-by doesn't exist, and some items are missing translation.

This work must be done item by item, line by line, because there are many missing features when it comes to accessibility.

-On several elements, the focus is not visible, for example on checkboxes, but more generally in input fields, often only the insertion cursor appears.
- Checkboxes all without labels, selects with autocomplete, often not correctly associated and with no translation;

**To Reproduce**
Simple use of "tab" and a screen reader inside the sidebar

**Expected behavior**
Check all labels of contextual sidebars, especially for checkboxes, and associate accessible 'aria-label' to select lists with search
Make the focus visible in all fields that receive it and require interaction.

**Screenshots**
It's a bit challenging to demonstrate this issue because what we need to observe is precisely what we're not seeing. I attempted to record a video starting the tab sequence from the beginning of the sidebar, but the focus occasionally vanishes (for instance, inside the objectBrowser field) or is non-existent, causing confusion.

https://github.com/plone/volto/assets/60133113/34a0b18c-8ce9-48f4-9422-39f0cc5751de

I did not register with the screen reader on because it might be confusing, as the screen reader creates a visual border around focused items.

**Software (please complete the following information):**
Volto Version 17.0.0-alpha.19
Plone Version 6.0.4

**Additional context**
There are many others issues for the sidebar, it's impossible to create just one issue to resolve all
